src/v2/Components/ArtworkFilter/ArtworkFilterContext.tsx

```diff
@@ -585,7 +591,8 @@ const artworkFilterReducer = (
 }
 
 export const getSelectedFiltersCounts = (
```

Comment:
Can we add a comment that describes `defaultFilters` works and maybe a `TODO: remove once A/B test is over` if this applies

Reply:
Fixed